server/handlers.go

```diff
@@ -215,8 +215,20 @@ func senderInfoHandler(client eth.LivepeerEthClient) http.Handler {
 
 		sender, err := client.Senders(client.Account().Address)
 		if err != nil {
-			respondWith500(w, fmt.Sprintf("could not query sender info: %v", err))
-			return
+			if err.Error() == "ErrNoResult" {
+				type Sender struct {
```

Comment:
Just curious, if the upstream Sender struct ever changes, will the compiler catch it here and prompt for an update?

Reply:
I think it does
<img width="926" alt="Screen Shot 2019-04-01 at 7 15 18 PM" src="https://user-images.githubusercontent.com/345743/55365422-87287c00-54b2-11e9-9d21-fce051aa74c2.png">